autolab/Autolab
# Passwordless temporary login

## Summary
This change allows an admin to set a user's password without needing the user's current password. 

The admin can also send the link to the user, the user can then use that link to reset his/her password without current password.

## Description
- Added a new action `reset_password` to the `UsersController` that resets a user's password and redirects to the password reset page.
- Changed the mode of the `User` model file from 100755 to 100644.
- Added a reset password link for administrators in the user show view.
- Added a new route for the `reset_password` action in the user resource.
- Made some linting changes to user.rb because the current version fails robocop

## How Has This Been Tested?
- Navigate to a user and click reset password
- Click to reset a user's password
- Try with different types of password and check that password verification works
- Type in a valid password (longer than 8 characters) that user's password has been reset and email sent
- Try to do the same by copying the link, but you have to be in incognito (signed out)

## Types of changes

## Checklist:
